### ISSUE TITLE
Jupyterlab blank screen after building latest master

### ISSUE THREAD

**Issue by lresende** [elyra-ai/elyra#1326]
**Describe the issue**
After building the latest master with Elyra extensions JupyterLab hangs during open with blank screen

**To Reproduce**
1. git pull --rebase
2. make clean install
3. jupyter lab --debug

![image](https://user-images.githubusercontent.com/382917/108797783-e5482780-7540-11eb-9acd-2453a4793ad3.png)

**Console log details**
```
Uncaught (in promise) TypeError: _interopRequireDefault is not a function
    at Object../node_modules/react-transition-group/node_modules/dom-helpers/class/addClass.js (addClass.js:8)
    at __webpack_require__ (bootstrap:18)
    at Object../node_modules/react-transition-group/CSSTransition.js (CSSTransition.js:8)
    at __webpack_require__ (bootstrap:18)
    at Object../node_modules/react-transition-group/index.js (index.js:3)
    at __webpack_require__ (bootstrap:18)
    at Module../node_modules/@blueprintjs/core/lib/esm/components/overlay/overlay.js (vendors-node_modules_blueprintjs_core_lib_esm_components_hotkeys_hotkeysDialog_js-node_module-431600.6ed3ea7cd2892fdd193f.js:8350)
    at __webpack_require__ (bootstrap:18)
    at Module../node_modules/@blueprintjs/core/lib/esm/components/popover/popover.js (vendors-node_modules_blueprintjs_core_lib_esm_components_hotkeys_hotkeysDialog_js-node_module-431600.6ed3ea7cd2892fdd193f.js:8886)
    at __webpack_require__ (bootstrap:18)
```

**Comment by lresende** [comment]
Originally from @akchinSTC and I confirmed I can reproduce with the latest master in the evening but was ok earlier in the day, so this might be one of those `dependency got a new release` things.

**Comment by lresende** [comment]
This was being caused by a [babel issue](https://github.com/babel/babel/issues/12854) that has been fixed in [babel 7.13.1](https://github.com/babel/babel/pull/12855)
